### PR TITLE
CC-19157 Moved the core code from `oryx`.

### PR DESCRIPTION
**Pull request by danilmoro3**
Ticket: https://spryker.atlassian.net/browse/CC-19157

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,30 +14,31 @@
...
+    "fast-glob": "~3.2.12",
     "file-loader": "~1.1.11",
-    "fs-extra": "~10.0.0",
+    "fs-extra": "~10.1.0",
     "imports-loader": "~0.8.0",
-    "mini-css-extract-plugin": "~0.4.0",
-    "optimize-css-assets-webpack-plugin": "~5.0.8",
-    "postcss-loader": "~2.1.4",
+    "mini-css-extract-plugin": "~2.7.2",
+    "moment": "~2.29.4",
```
Comment (tolerants) [inline package.json]: Suggest to replace huge momentjs dependency with something smaller - [dayjs](https://github.com/iamkun/dayjs) for example.

File: package.json [hunk 2]
```diff
@@ -14,30 +14,31 @@
...
+    "npm": ">=8.0.0"
   },
   "dependencies": {
-    "@babel/core": "~7.17.12",
-    "@babel/preset-env": "~7.17.12",
-    "@spryker/oryx": "^1.4.0",
-    "autoprefixer": "~8.3.0",
-    "babel-loader": "~8.2.5",
-    "core-js": "~2.5.5",
+    "@babel/core": "~7.21.3",
```
Comment (tolerants) [inline package.json]: Also it's probably a good time to move dev dependencies into `devDependencies` section.

**Review by tolerants — CHANGES_REQUESTED** [review]

**Review by tolerants — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,7 +1,5 @@
 # Oryx for Zed
 
-Spryker ZED frontend automation tool (Oryx based)
+Spryker ZED frontend automation tool
 
 > Our documentation has moved. Please go to [our official Spryker documentation](https://documentation.spryker.com/front-end_developer_guide/zed/oryx/oryx-for-zed.htm).
```
Comment (FRITS-Kh) [inline README.md]:
```suggestion
The tool [documentation](https://docs.spryker.com/docs/scos/dev/front-end-development/zed/oryx-for-zed.html).
```

File: lib/configure-paths.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                import: value,
+            };
+        }
+
+        if (key !== runtimeEntryName) {
+            dependedEntryPaths[key] = {
+                dependOn: runtimeEntryName,
+                import: value,
+            };
+        }
```
Comment (FRITS-Kh) [inline lib/configure-paths.js]:
```suggestion
            };
            
            return;
        }
        
        dependedEntryPaths[key] = {
            dependOn: runtimeEntryName,
            import: value,
        };
```

File: lib/find.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        const name = defineName ? defineName(currentPath) : path.basename(currentPath);
+
+        object[name] = currentPath;
+
+        return object;
+    }, {});
+}
+
+async function find(settings, initial) {
+    settings = parseSettings(settings);
```
Comment (FRITS-Kh) [inline lib/find.js]: Let's not modify the parameters.

File: lib/find.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+function parseSettings(settings) {
+    return Object.assign({}, {
+        glob: { onlyFiles: false },
+    }, settings);
+}
+
+function parseOrigin(initial) {
+    return initial || {};
+}
```
Comment (FRITS-Kh) [inline lib/find.js]: Let's have the empty object as a default value of the parameter instead of the method.

File: package.json [hunk 7]
```diff
@@ -1,43 +1,46 @@
...
-    "@babel/core": "~7.17.12",
-    "@babel/preset-env": "~7.17.12",
-    "@spryker/oryx": "^1.4.0",
-    "autoprefixer": "~8.3.0",
-    "babel-loader": "~8.2.5",
-    "core-js": "~2.5.5",
+    "autoprefixer": "~9.8.8",
+    "babel-loader": "~8.3.0",
+    "colors": "~1.4.0",
+    "core-js": "~2.6.12",
```
Comment (FRITS-Kh) [inline package.json]: Can we start using the `core-js` v3?
If we still need it due to stop support of IE11.

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: package.json [hunk 8]
```diff
@@ -1,43 +1,46 @@
...
-    "@babel/core": "~7.17.12",
-    "@babel/preset-env": "~7.17.12",
-    "@spryker/oryx": "^1.4.0",
-    "autoprefixer": "~8.3.0",
-    "babel-loader": "~8.2.5",
-    "core-js": "~2.5.5",
+    "autoprefixer": "~9.8.8",
+    "babel-loader": "~8.3.0",
+    "colors": "~1.4.0",
+    "core-js": "~3.29.1",
```
Comment (FRITS-Kh) [inline package.json]: Please check if the `core-js` is still necessary.

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: package.json (hunk 8 above)
Comment (danilmoro3) [reply package.json]: Removed

**Review by FRITS-Kh — APPROVED** [review]

**Pull request merged**